biz.aQute.launcher/src/aQute/launcher/plugin/ProjectLauncherImpl.java

```diff
@@ -260,6 +261,11 @@ public Jar executable() throws Exception {
 					runpathShas.add(sha + ";name=\"" + file.getName() + "\"");
 				} else {
 					String newPath = "jar/" + file.getName();
+					String[] parts = Strings.extension(file.getName());
+					int i = 1;
+					while (jar.exists(newPath)) {
+						newPath = String.format("%s[%d].%s", parts[0], i++, parts[1]);
```

Comment:
Doesn't this lose the `jar/` prefix? Shouldn't the format string be `"jar/%s[%d].%s"`?